torch/onnx/_internal/exporter/_torchlib/ops/nn.py

```diff
@@ -48,8 +48,10 @@ def aten_group_norm(
 
     c = op21.Shape(input, start=1, end=2)
     if weight is None:
+        # pyrefly: ignore [missing-attribute]
```

Comment:
errors for this file were ignored with https://github.com/maggiemoss/pytorch/blob/5378dc52cb0cf501542e7e1af638be1c71470d0f/torch/onnx/_internal/exporter/_torchlib/ops/nn.py#L3. Could you help check if the directive was written correctly and remove all of the inline ignores? Thanks